assignment/pradeepk/session9/app.js

```diff
@@ -1,71 +0,0 @@
-
-// Reverse A String
-
-var str = 'pradeep';
-var strReverse = str.split('').reverse().join('');
-console.log(strReverse);
-
-
-//palindrome
-
-var str = 'mom';
-var strReverse = str.split('').reverse().join('');
-if(str===strReverse){
-    console.log("yes");
```

Comment:
do not delete some one else file, do git status and check before adding and deleting file, you can do git checkout . or undo this commit